test/controllers/roles_controller_test.rb

```diff
@@ -0,0 +1,29 @@
+require "test_helper"
+
+describe RolesController do
+  include SearchApiHelpers
+
+  def ministerial_role_json
+    json = File.read("test/fixtures/content_store/ministerial_role.json")
```

Comment:
Do you think this might be better as a [content-schema example](https://github.com/alphagov/govuk-content-schemas/tree/master/examples)? (I appreciate the prior art in this repo...)

Reply:
Yep good idea. So I would need to make an example of [this](https://github.com/alphagov/govuk-content-schemas/blob/25cad7a436763199272507b8ac8e4d80da03ac1b/formats/role.jsonnet)?

Reply:
https://github.com/alphagov/govuk-content-schemas/pull/1049